src/app/Instance/InstancePage/ProcessorsTabContent.tsx

```diff
@@ -142,6 +137,20 @@ export const ProcessorsTabContent = ({
     error: processorsError,
   } = useGetProcessorsApi();
 
+  const customToolbarElement = areProcessorsLoading ? (
+    <Skeleton width={"170px"} height={"35px"} />
+  ) : bridgeStatus == ManagedResourceStatus.Failed ? (
```

Comment:
There is a tiny time interval when `bridgeStatus` is `undefined` and the "Create processor" button is enabled.
Maybe a "better" condition for enabling the processor creation could be `bridgeStatus !== ManagedResourceStatus.Ready`?